mesh/submesh/psubmesh.hpp

```diff
@@ -64,7 +68,7 @@ class ParSubMesh : public ParMesh
     * @param[in] domain_attributes Domain attributes to extract
     */
    static ParSubMesh CreateFromDomain(const ParMesh &parent,
-                                      Array<int> &domain_attributes);
+                                      const Array<int> &domain_attributes);
 
    /**
    * @brief Create a surface ParSubMesh from it's parent.
```

Comment:
`it's` -> `its` (preexisting typo) in 2 places.

Reply:
Fixed in https://github.com/mfem/mfem/pull/4476/commits/14e511403985758e7c9c8c131907bcad1a3a995b